easybuild/easyconfigs/p/ProbABEL/ProbABEL-0.5.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'ProbABEL'
+version = '0.5.0'
+
+homepage = "http://www.genabel.org/packages/ProbABEL"
+description = """Tool for genome-wide association 
+analysis of imputed genetic data."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.genabel.org/sites/default/files/software/']
+sources = ['probabel-%(version)s.tar.gz']
+
+dependencies = [
+    ('Eigen', '3.2.6')
+]
+
+configopts = '--disable-latex-doc'
+
+postinstallcmds = ["mv %(installdir)s/etc/probabel_config.cfg.example %(installdir)s/etc/probabel_config.cfg"]
+
+sanity_check_paths = {
+    'files': ['bin/probabel', 'bin/pacoxph',
+	'bin/palinear', 'bin/palogist','etc/probabel_config.cfg.example'],
```

Comment:
@rtripath89 file was renamed, so last entry here should be `etc/probabel_config.cfg` to make the sanity check pass (which probably means you can collapse this line with the line above as well)